src/index.js

```diff
@@ -0,0 +1,58 @@
+import './style.css';
+
+const todo = [
+  { description: 'Go Bazar', isCompleted: false, index: 1 },
+  { description: 'Go to the gym', isCompleted: false, index: 2 },
+  { description: 'Go to the Field', isCompleted: false, index: 3 },
+];
+
+const mainContainer = document.querySelector('#todo-container');
+const populateTodo = (todo) => {
```

Comment:
Please make sure the tasks are rendered in order of the `index` values for each task.
![list](https://user-images.githubusercontent.com/35504149/200016406-8d18d508-20fd-4c64-a894-925652f514c5.png)

![list1](https://user-images.githubusercontent.com/35504149/200016433-39080204-3aef-4ad7-bf3d-46d06d0a4c12.png)